features/steps/step_create_emoticons.py

```diff
@@ -0,0 +1,25 @@
+from behave import given, when, then
+
+@given("we are on Hipchat Emoticons Page")
+def step_impl(context):
+    context.authorized_page.switch_to_emoticons()
+
+@when('we go to emoticons constructor')
+def step_impl(context):
+    context.emoticons_page.go_to_constructor()
+
+@when('we upload a picture')
+def step_impl(context):
+    context.emoticons_page.upload_image()
+
+@when('we click "Add emoticon"')
+def step_impl(context):
+    context.emoticons_page.click_add_emoticon()
+
+@then('we see new emoticon')
+def step_impl(context):
+    context.emoticons_page.check_if_created()
```

Comment:
I don't see where you check that you create, I see that you try to find this element, but not check. I think that you should return True, if element present, and False if is not.
Maybe we should to ask help in this moment, I'm not sure that I'm right.

Reply:
Replace to wait_if_created. We are waiting for visibility of creation a new emoticon.